src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -654,11 +741,43 @@ export default {
 }
 
 .conversations-search {
-	flex-grow: 1;
+	transition: all 0.3s ease;
+	width: calc(65% - 8px);
+	z-index: 1;
+	position : absolute;
 
 	:deep(.input-field__input) {
 		border-radius: var(--border-radius-pill);
 	}
+	&--expanded {
+		width: 90%;
+	}
+
+}
+
+//FIXME : this should be changed once the disabled style for input is added
+:deep(.input-field__input[disabled="disabled"]){
+	background-color: var(--color-background-dark);
+}
+
+.options{
+	position: relative;
+	transition: all 0.3s ease; //gets hidden once the search box is expanded
+	left : calc(65% + 4px);
+	display: flex;
+}
```

Comment:
```suggestion
  height: var(--default-clickable-area);
}
```
let's do it like this, because now i see in devtools both yours .hidden-visually and the one from server (overlapping styles)